snappy/snap_remote.go

```diff
@@ -34,7 +34,8 @@ import (
 // RemoteSnap represents a snap available on the server
 type RemoteSnap struct {
 	// FIXME: exported for testing
-	Pkg remote.Snap
+	Pkg             remote.Snap
+	currentCurrency string
```

Comment:
Hmm, this makes me feel uneasy. In what situation would we have multiple currencies in the system?